packages/backend/src/modules/dashboard/dashboard.model.ts

```diff
@@ -0,0 +1,39 @@
+import mongoose from 'mongoose'
+import { DashboardVisibility } from 'shared'
+
+const { ObjectId } = mongoose.Types
+
+const columnSchema = new mongoose.Schema({
+  _id: ObjectId,
+  title: String,
+  tasks: [
+    {
+      _id: ObjectId,
+      title: String,
+      imageCoverId: Number,
+      comments: [{ _id: ObjectId }],
+      attachments: [{ _id: ObjectId }],
+      users: [{ _id: ObjectId }]
+    }
+  ]
+})
+
+const dashboardSchema = new mongoose.Schema({
+  _id: ObjectId,
+  title: String,
+  description: String,
+  imageCoverUrl: String,
+  createdAt: Date,
+  status: DashboardVisibility,
```

Comment:
is it working? I'vs got:
trello-backend: TypeError: Invalid schema configuration: `PUBLIC` is not a valid type at path `status.PUBLIC`. See https://bit.ly/mongoose-schematypes for a list of valid schema types.